tests/get_external_dependencies.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import os
+import sys
+
+from utils import read_dependencies
+
+distributions = sys.argv[1:]
+
+if not distributions:
+    distributions = os.listdir("stubs")
+
+for distribution in distributions:
+    for package in read_dependencies(distribution).external_pkgs:
+        print(f"{package} ")
```

Comment:
Why the f-string here?

Reply:
For the space.
And the space was needed (at least locally) or it'd store something like `numpycomtypes` in the variable.
Might just be a weird windows bash issue though

Reply:
Would this work?

```suggestion
        print(package, end=" ")
```

Reply:
yeah ok that's it, works fine in powershell, I'll change this to no space